crates/turbo-tasks/src/value_type.rs

```diff
@@ -111,7 +100,7 @@ impl ValueType {
 
     /// This is internally used by `#[turbo_tasks::value]`
     pub fn new_with_magic_serialization<
-        T: Debug + Eq + Ord + Hash + Serialize + for<'de> Deserialize<'de> + Send + Sync + 'static,
+        T: Debug + Eq + Hash + Serialize + for<'de> Deserialize<'de> + Send + Sync + 'static,
```

Comment:
Outside the scope of this PR, but the complicated and often-repeated bounds here might be a good candidate for the "trait alias" trick:

https://www.reddit.com/r/rust/comments/1582jku/how_can_one_create_a_trait_alias/

```
trait Valid: PartialOrd + Display + PartialEq + Clone {}
impl<T: PartialOrd + Display + PartialEq + Clone> Valid for T {}
```

("quotes" around "trait alias" because real trait aliases aren't stable)